open_in_mpv/open_in_mpv.py

```diff
@@ -4,19 +4,36 @@
 from typing import Any, BinaryIO, Callable, Final, Mapping, TextIO, cast
 import json
 import os
+import random
 import socket
 import struct
 import subprocess as sp
 import sys
 import tempfile
+import time
 
 from loguru import logger
 import click
 import xdg.BaseDirectory
 
 from .constants import IS_MAC, IS_WIN, MACPORTS_BIN_PATH
 
+try:
+    from win32file import GENERIC_READ, GENERIC_WRITE, OPEN_EXISTING, CreateFile, WriteFile
+    from win32pipe import PIPE_READMODE_MESSAGE, SetNamedPipeHandleState  # cspell:disable-line
+    import pywintypes  # cspell:disable-line
+except ImportError:
+    if IS_WIN:
+        logger.error('Failed to resolve pywin32. Exiting...') # cspell:disable-line
+        os._exit(os.EX_SOFTWARE) # pylint: disable=protected-access
```

Comment:
Why call `_exit()` here instead of normal exit?

Reply:
took the same approach as `spawn` though this can be changed to `os.exit` if preferred.